components/x-gift-article/src/lib/variables.scss

```diff
@@ -0,0 +1,3 @@
+// This is needed for calls to the image service for Icons used in Social,
+// and oMessage
+$system-code:'github:Financial-Times/x-dash';
```

Comment:
is there a way we can set this only be included in the tests, so that apps don't forget to set it?

Reply:
Had a look at this but I can't work out how to do it such that it works for storybook and the tests. Going to open an issue for this and move on as I agree it is the right thing to do but it doesn't need to block this PR.